### PR TITLE
Fix issue-13952-double-click-edit-button-duplicate-edit-controls

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts [hunk 1]
```diff
@@ -512,5 +509,33 @@ describe('DotEditPageToolbarComponent', () => {
             clickStateButton('preview');
             expect(component.lockerModel).toBe(true, 'page locked after click in preview');
         });
+
+        it("should edit tab don't be called twice", () => {
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts]: Single quotes... looks like something break your prettier.

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts [hunk 2]
```diff
@@ -224,17 +221,17 @@ describe('DotEditPageToolbarComponent', () => {
         const lockedMessage: DebugElement = de.query(By.css('.edit-page-toolbar__locked-by-message'));
         expect(lockedMessage.nativeElement.textContent).toContain('Page is locked');
 
-        const editStateModel = component.states.find(state => state.label === 'Edit');
+        const editStateModel = component.states.find((state) => state.label === 'Edit');
         expect(editStateModel.styleClass).toEqual('edit-page-toolbar__state-selector-item--disabled');
     });
 
-    it('should have page can\'t edit message and disabled edit button', () => {
+    it("should have page can't edit message and disabled edit button", () => {
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts]: Single quotes... looks like something break your prettier.

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts [hunk 3]
```diff
@@ -26,6 +27,8 @@ export class DotEditPageToolbarComponent implements OnInit, OnChanges {
     lockerModel: boolean;
     mode: PageMode;
 
+    private debounceStateSelector = _.debounce((pageState: PageMode) => this.setSelectorState(pageState), 500, { leading: true });
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts]: 500ms? you sure? that looks too slow to wait for an user when click something.

Comment (alfredo-dotcms) [reply src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts]: on the test I did it looked good to me! what do you think would be a proper ms?

Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts]: Nah... all good I tested it and seems to be working fine.

**Pull request merged**